packages/cloudflare/src/cli/build/patches/to-investigate/wrangler-deps.ts

```diff
@@ -23,6 +26,8 @@ export function patchWranglerDeps(config: Config) {
 
   writeFileSync(pagesRuntimeFile, patchedPagesRuntime);
 
+  patchRequireReactDomServerEdge(config);
```

Comment:
q: should this be a different patch so that we can better assert if it's (or the existing patchWranglerDeps) patch is applied?

Reply:
this patches the `.next/standalone/node_modules/next/dist/compiled/next-server/pages.runtime.prod.js` file, which is already being patched by the `patchWranglerDeps` function, so I tough it was correct to include it here, I don't have any preference on where to include this logic this just seemed the most natural and consistent place right now

I'm actually not sure, `patchWranglerDeps` patches multiple unrelated things/files, shouldn't those also be their own separate patch functions?

Reply:
@vicb I'm happy to merge but please let me know about the above, I am happy with whatever solution you prefer 🙂

Reply:
Merge it 🚀
We can always update later if needed